usmqe/usmmail.py

```diff
@@ -0,0 +1,91 @@
+"""
+Module for working with email alerts.
+
+.. moduleauthor:: ebondare
+
+Usage::
+
+    import usmqe.usmmail
+
+    # ...get all  messages from /var/mail/root on the client machine
+    # Return a string
+
+    messages = usmqe.usmmail.get_client_mail()
+
+    # ...get messages from /var/mail/root on the client machine
+    # that came within the specified time interval
+    # Return a mailbox object
+
+    messages = usmqe.usmmail.get_msgs_by_time(start_timestamp, end_timestamp)
+"""
+
+
+import pytest
+import usmqe.usmssh
+import mailbox
+import time
+import email.utils
+import os
+import tempfile
+
+
+LOGGER = pytest.get_logger('usmmail', module=True)
+
+
+def create_mailbox_file(filename='mbox_file', content=''):
+    tmpdirname = tempfile.mkdtemp()
+    mbox_file_path = os.path.join(tmpdirname, filename)
+    with open(mbox_file_path, 'w') as f:
+        f.write(content)
+    return mbox_file_path
+
+
+def get_client_mail():
+    """
+    Read mail from /var/mail/root on client machine.
+    Return the contents of the mailbox as a string
+    """
+    SSH = usmqe.usmssh.get_ssh()
+    host = usmqe.inventory.role2hosts("usm_client")[0]
+    cat_mail_log_cmd = "cat /var/mail/root"
+    retcode, stdout, stderr = SSH[host].run(cat_mail_log_cmd)
+    LOGGER.debug("Return code of 'cat /var/mail/root': {}".format(retcode))
+    LOGGER.debug("Stderr of cat: ".format(stderr.decode()))
+    if retcode != 0 and stderr.decode().count('No such file') > 0:
+        return ''
+    if retcode != 0:
+        raise OSError(stderr)
+    return stdout.decode()
+
+
+def get_msgs_by_time(start_timestamp=None, end_timestamp=None):
```

Comment:
I would add parameters `host` and `user` for mail client that could have default values loaded from configuration but that can be overwritten from testcase. This might be part of next pull request.